rust/private/repository_utils.bzl

```diff
@@ -396,13 +405,17 @@ def load_rust_stdlib(ctx, target_triple):
     if "BAZEL_RUST_STDLIB_LINKFLAGS" in ctx.os.environ:
         stdlib_linkflags = ctx.os.environ["BAZEL_RUST_STDLIB_LINKFLAGS"].split(":")
 
+    rustc_srcs = None
+    if should_include_rustc_srcs(ctx):
+        rustc_srcs = "\"@rust-src_{}//:rustc_srcs\"".format(ctx.attr.version)
```

Comment:
Hard-coding the name here seems a little strange - would it make more sense to have a new attr for the name of the `rust-src` repository (or full target) which may be `None`, and set `rustc_srcs` according to it, and having `rust_repository_set` set that attribute as it expects, rather than hard-coding what a wrapper happens to set the name to?